frontend/components/panel/js/panel.component.ts

```diff
@@ -214,9 +214,7 @@ class PanelController implements ng.IController {
 	public calculateContentHeights() {
 		const tempContentItemsShown = angular.copy(this.contentItemsShown);
 		this.assignHeights(this.maxHeightAvailable, tempContentItemsShown, null);
-		this.$timeout(() => {
-			this.$scope.$apply();
-		});
+		//this.$timeout(() => {});
```

Comment:
remove unless there is a reason why it needs to stay (in which case we need a comment here)